dali/test/python/test_utils.py

```diff
@@ -32,20 +32,23 @@ def check_batch(batch1, batch2, batch_size, eps = 1e-07):
         is_failed = False
         assert(batch1.at(i).shape == batch2.at(i).shape), \
             "Shape mismatch {} != {}".format(batch1.at(i).shape, batch2.at(i).shape)
-        try:
-            err = np.mean( np.abs(batch1.at(i) - batch2.at(i)) )
-        except:
-            is_failed = True
-        if (is_failed or err > eps ):
+        assert(batch1.at(i).size == batch2.at(i).size), \
+            "Size mismatch {} != {}".format(batch1.at(i).size, batch2.at(i).size)
+        if batch1.at(i).size != 0:
```

Comment:
```suggestion
        assert batch1.at(i).size > 0
```

Reply:
Intention here was that it might make sense for `batch1.size()` to be zero: like no objects in the image. In such a case code in `try` block gives some weird warnings. At this point we now that sizes are equal, so if they are both `0` it is fine.